### PR TITLE
TS-367: ignore webhook if order is cancelled

### PR DESCRIPTION
**Pull request by tomas-maciulis**

### CODE REVIEW

File: translations/en.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
 $_MODULE['<{kevin}prestashop>kevin_8a5f368eac3aab699cf8e6e1ec28b695'] = 'Please enter an amount to proceed with your refund.';
 $_MODULE['<{kevin}prestashop>kevin_c560cbc6f74c01a7ec17e2752afeee45'] = 'Refund process has started. Amount';
 $_MODULE['<{kevin}prestashop>kevin_39750b492c150b85fbb3e8ca510f6168'] = 'Refund has failed.';
-$_MODULE['<{kevin}prestashop>kevin_efc9732522e9b2e2718488950ef95e2b'] = 'Payment has Started';
+$_MODULE['<{kevin}prestashop>kevin_efc9732522e9b2e2718488950ef95e2b'] = 'Payment has started';
 $_MODULE['<{kevin}prestashop>kevin_3e984424a2dcd938412cace671b40e59'] = 'Payment is Pending';
```
Comment (saltanovas) [inline translations/en.php:34]: Perhaps lowercase `Pending` then as well?

**Review by saltanovas — CHANGES_REQUESTED** [review]

**Review by saltanovas — APPROVED** [review]

**Comment by saltanovas** [comment]
Do we need QA here? :D I would say just merge

**Comment by xpresas** [comment]
Couldn't wait for you guys to merge.

**Comment by saltanovas** [comment]
@xpresas we always do QA now. If you need features from a specific branch, you should checkout that specific feature branch and open a pull request on it.

We will try to do define some restrictions in the feature.

**Comment by saltanovas** [comment]
If you have in mind manual tests from developer side, then yes, and it is extremely valuable :) But we do have internal dedicated people to ensure the overall plugin quality

**Pull request merged**